<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualSegmentationDockWidget</class>
 <widget class="QDockWidget" name="ManualSegmentationDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>134</width>
    <height>258</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>123</width>
    <height>237</height>
   </size>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Manual Segmentation</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="OnRBtn">
          <property name="text">
           <string>On</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="OffRBtn">
          <property name="text">
           <string>Off</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="SettingsBtn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Settings...</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="ReinitializeBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Reinit.</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+R</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ValidateBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Validate</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+V</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ReinitializeIncrementBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Reinit. + Incr.</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+I</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>OffRBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>SettingsBtn</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OffRBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>ReinitializeBtn</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OffRBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>ValidateBtn</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OffRBtn</sender>
   <signal>toggled(bool)</signal>
   <receiver>ReinitializeIncrementBtn</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
